### ISSUE TITLE
Install overhead kitchen cupboards

### ISSUE THREAD

**Issue by zarino** [DoESLiverpool/somebody-should#770]
Broken out from #690.

Affix the black, overhead kitchen cupboards onto the plasterboard wall to your immedate left as you enter the kitchen (where the bin currently is) – in either a 2x2 configuration at chest-to-head height, or in a tall 1x4 "tower" configuration, depending how much space there is.

**Comment by johnmckerrell** [comment]
This depends somewhat on where the studs are in the wall. It might be we could put 2 on this wall and 2 on the opposite wall? Or maybe 2 on this wall and retire the other 2.

**Comment by magman2112** [comment]
I looked at these cupboards last week. Is there any associated mounting hardware saved somewhere for these cabinets, as there appear to just be some fixture holes on the back with no adjustment, which is unusual for kitchen cabinets?

Also, I would suggest putting these cabinets in a 2x2 arrangement on the wall next to the hatch into the Event space, we could then use @jackie1050 suggestion that we make shelves for the other wall, which would potentially make better use of the space, but this would also need a bit of thought and a design.

**Comment by johnmckerrell** [comment]
Each hole should have a vaguely corrugated plate to go with it, I thought we had most of them perhaps missing just 2? Perhaps you got unlucky with the cupboard you looked in? 2x2 Does somewhat seem to make sense although the higher ones will be very high.

**Comment by amcewen** [comment]
@magman2112 has installed the cupboards, but AIUI there's still something to finish off, I'm sure he'll update us presently

**Comment by magman2112** [comment]
I managed to source the correct mounting brackets from eBay. 

After further discussion, the cabinets have been installed on the opposite wall to that which @zarino originally discussed. Strips of plywood where fixed to the wall first, then the cabinets attached to this plywood.

All 4 cabinets have been fixed to the wall, but only 3 doors have been fitted so far. We appear to have lost 3 of the screws for fixing the hinge to the last cabinet, so I’m trying to find replacements so that I can complete this task.
